types/slickgrid/index.d.ts

```diff
@@ -93,16 +93,16 @@ declare namespace Slick {
 		* @method subscribe
 		* @param fn {Function} Event handler.
 		*/
-		public subscribe(fn: (e: EventData, data: T) => any): void;
-		public subscribe(fn: (e: DOMEvent, data: T) => any): void;
+		public subscribe(fn: (e: EventData, data: T) => void): void;
+		public subscribe(fn: (e: DOMEvent, data: T) => void): void;
```

Comment:
Typing a callback's return value as `void` is the best practice: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#use-optional-parameters